src/components/searchbar/SearchBar.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect } from "react";
+import './SearchBar.scss';
+import TextField from '@material-ui/core/TextField';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import SearchIcon from '@material-ui/icons/Search';
+
+export default function ColorTextFields() {
+  return (
+    <form noValidate autoComplete="off">
+      <TextField
+        id="outlined-secondary"
```

Comment:
use id's for identifying components when searching, etc. But i think you're just styling this component so stick with className